### PR TITLE
Samples Post Release Tasks

### PR DESCRIPTION
**Pull request by shubham7109**

### CODE REVIEW

**Comment by shubham7109** [comment]
Ignoring build checker failure since it is reading the main readme file.

**Review by hud10837 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-ArcGIS Maps SDK for Kotlin v200.2.0 samples.  The `main` branch of this repository contains sample app modules for the latest available version of the [ArcGIS Maps SDK Android Kotlin](https://developers.arcgis.com/kotlin/). Samples released under older versions can be found through the [git tags](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/tags).  Please read our [wiki](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/wiki) for help with working with this repository.
+ArcGIS Maps SDK for Kotlin v200.3.0 samples.  The `main` branch of this repository contains sample app modules for the latest available version of the [ArcGIS Maps SDK Android Kotlin](https://developers.arcgis.com/kotlin/). Samples released under older versions can be found through the [git tags](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/tags).  Please read our [wiki](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/wiki) for help with working with this repository.
```
Comment (gunt0001) [inline README.md:5]: Shouldn't it say 200.4.0 ?
```suggestion
ArcGIS Maps SDK for Kotlin v200.4.0 samples.  The `main` branch of this repository contains sample app modules for the latest available version of the [ArcGIS Maps SDK Android Kotlin](https://developers.arcgis.com/kotlin/). Samples released under older versions can be found through the [git tags](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/tags).  Please read our [wiki](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/wiki) for help with working with this repository.
```

File: version.gradle [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ext {
```
Comment (prupani-7) [inline version.gradle:1]: While reviewing the other PR https://devtopia.esri.com/runtime/kotlin/pull/3392/files, I was verifying myself to see if samples, toolkit and kotlin repo versions.gradle match. came across this PR
I noticed, below three version that need to be updated to match the other version upgrades?
   ```
 androidBrowserVersion = '1.5.0'
    // Plugin versions
    gradleVersion = '8.1.1'
    // Java version
    javaVersion = 1.17
```
to 
   ```
 androidBrowserVersion = '1.7.0'
    // Plugin versions
    gradleVersion = '8.2.0'
    // Java version
    javaVersion = 1.8
```

**Comment by shubham7109** [comment]
@prupani-7 Let's update dependencies in a separate PR once the SDK changes are added. This PR is to allow for daily builds to run.

**Review by prupani-7 — APPROVED** [review]
Looks good 👍

**Pull request merged**